src/lib/prof-lean/crypto-hash.h

```diff
@@ -159,6 +159,10 @@ crypto_hash_self_test
   int verbose
 );
 
+#if defined(__cplusplus)
+}
+#endif
+
```

Comment:
Unrelated code region, would break the build if self-test was switched on. Remove.